.github/workflows/format.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: 3.11
```

Comment:
pyproject-flake8 doen't support python3.12. So, I use Python3.11 for format CI.

https://github.com/csachs/pyproject-flake8/issues/30